package.json

```diff
@@ -20,6 +20,7 @@
     "astro": "4.15.1",
     "semver": "7.6.3",
     "sharp": "0.33.5",
+    "@alexanderniebuhr/astro-monthly-blog-resource-loader": "0.0.1",
```

Comment:
Is there a reason not to have the loader inside the repo directly? It doesn't have to be a package right?

Reply:
I guess it could be added inline too, but I'm not sure if I want to bloat this repository with that and have to maintain it here. But whatever you think is better, is fine I guess.

Reply:
I don't have strong opinions on this, I think maybe it would be easier to maintain inline since it's just intended for internal use. I let you make the final call on that!